plexpy/__init__.py

```diff
@@ -182,6 +182,7 @@ def initialize(config_file):
         if CONFIG.PMS_TOKEN and CONFIG.PMS_IP and CONFIG.PMS_PORT:
             plextv.get_real_pms_url()
             pmsconnect.get_server_friendly_name()
+            pmsconnect.PmsConnect().make_meta_data_cache
```

Comment:
sry, copy paste error from schduler, this is also the reason you had to manually refresh it the first time.